src/main/java/com/carhouse/consumers/WebConsumer.java

```diff
@@ -0,0 +1,17 @@
+package com.carhouse.consumers;
+
+import java.sql.Date;
+import java.util.List;
+
+/**
+ * The interface for data provider.
+ */
+public interface WebConsumer {
```

Comment:
What kind of information does this interface provide?

Reply:
I'll add it